test.html

```diff
@@ -12,12 +12,13 @@
     <title>LinkedIn UI Clone</title>
   </head>
   <body>
+    <div class="loading"></div>
     <div class="header">
       <div class="header__left">
-        <img
-          src="https://www.flaticon.com/free-icon/clown_9254829?related_id=9254829&origin=pack"
+        <a href="http://localhost:1234"><img
```

Comment:
so that localhost:1234 only exists when your server is running, can you change this to a different URL?